glue/app/qt/application.py

```diff
@@ -733,11 +737,11 @@ def _reset_session(self, show=True):
 
         if not os.environ.get('GLUE_TESTING'):
             buttons = QMessageBox.Ok | QMessageBox.Cancel
-            dialog = QMessageBox.warning(self, "Confirm Close",
-                                         "Are you sure you want to reset the session? "
-                                         "This will close all datasets, subsets, and data viewers",
-                                         buttons=buttons,
-                                         defaultButton=QMessageBox.Cancel)
+            dialog = QMessageBox.warning(
+                self, "Confirm Close",
+                "Are you sure you want to reset the session? "
+                "This will close all datasets, subsets, and data viewers",
+                buttons=buttons, defaultButton=QMessageBox.Cancel)
```

Comment:
In general, I prefer not to change the line length in cases like this when it actually makes the code less readable - would you mind disabling the line length PEP8 warnings or changing the line length to 100 or 120?